packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2669,7 +2672,8 @@ class InputDecoration {
        enabledBorder = null,
        semanticCounterText = null,
        alignLabelWithHint = false,
-       constraints = null;
+       constraints = null,
+       error = null;
```

Comment:
The `error` field should go just before `errorText`: here, in all the other lists where it appears, and in the field's definition.

That keeps things organized logically, and follows the pattern of `label`/`labelText`, `prefix`/`prefixText`, and others. See style guide:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#order-other-class-members-in-a-way-that-makes-sense

Reply:
nit: ordering here still